tgui/packages/tgui/interfaces/NanoCrucible.tsx

```diff
@@ -0,0 +1,95 @@
+/**
+ * @file
+ * @copyright 2025
+ * @author FlameArrow57 (https://github.com/FlameArrow57)
+ * @license ISC
+ */
+
+import { Button, Flex, Icon, Section } from 'tgui-core/components';
+
+import { useBackend } from '../backend';
+import { Window } from '../layouts';
+
+interface PacketInfo {
+  first_part;
+  first_part_img;
+  second_part;
+  second_part_img;
+  result_part;
+}
+
+export const NanoCrucible = () => {
+  const { act, data } = useBackend<PacketInfo>();
+  const {
+    first_part,
+    first_part_img,
+    second_part,
+    second_part_img,
+    result_part,
+  } = data;
+  return (
+    <Window title="Nano-Crucible" width={300} height={125}>
+      <Window.Content>
+        <Section fill>
+          <Flex align="center" textAlign="center" justify="space-between">
+            <Flex.Item grow>
+              <Button
+                onClick={() => act('load_first_part')}
+                tooltip={first_part}
+                color={!first_part ? 'orange' : 'green'}
```

Comment:
This should use standard colours, try to match any other interface that has a "click to insert" button, see chem dispensers etc.